tests/integration/extract_test.py

```diff
@@ -0,0 +1,22 @@
+"""Validate basic 10k and exhibit 21 extraction."""
+
+import unittest
+
+from mozilla_sec_eia.extract import (
+    _get_experiment_name,
+    _get_most_recent_run,
+    validate_extraction,
+)
+
+
+def test_basic_10k_extraction(test_mlflow_init_func):
+    """Run full 10k extraction on validation set and verify desired metrics are met."""
+    with unittest.mock.patch("mozilla_sec_eia.extract.initialize_mlflow"):
```

Comment:
I haven't used `unittest.mock.patch` before, what does this do to initiate the test?

Reply:
This just stops `validate_extraction` from actually calling `initialize_mlflow`, so I can configure mlflow with test settings instead of it trying to use the actual tracking server. Instead, when it tries to call `initialize_mlflow` it will call a `Mock` class that doesn't do anything, but will log that it was called and what the args were if you want to check for a test.